content-src/styles/_variables.scss

```diff
@@ -29,6 +29,7 @@ $grey-10-95: rgba($grey-10, 0.95);
 $grey-20-60: rgba($grey-20, 0.6);
 $grey-20-80: rgba($grey-20, 0.8);
 $grey-30-60: rgba($grey-30, 0.6);
+$grey-40-36: rgba($grey-40, 0.36);
```

Comment:
Is this to match comps?

Reply:
Yes.

Reply:
I doubt that's to photon spec, but I guess we can get updated designs later.